### ISSUE TITLE
Is the projected maintained anymore?

### ISSUE THREAD

**Issue by flexboom** [php-decimal/extension#84]
Can I contribute? I would like to help if I can.

**Comment by rtheunissen** [comment]
Yes it is. There is a v2 branch that is feature complete, but the docs still cover v1. I need to update the docs, then release v2 properly. Step one would be to rebase v2 branch on main.

**Comment by rtheunissen** [comment]
Yes! v1.5.3 and v2.0.0 are now released.
